backend/pom.xml

```diff
@@ -218,7 +232,7 @@
 	</dependencies>
 
 	<build>
-		<finalName>modeldb-${version}-client-build</finalName>
+		<finalName>modeldb-${project.version}-client-build</finalName>
```

Comment:
when running `mvn package -Dmaven.test.skip=true` I got this warning in the output:

```
[WARNING]
[WARNING] Some problems were encountered while building the effective model for ai.verta.modeldb:modeldb:jar:1.0-SNAPSHOT
[WARNING] The expression ${version} is deprecated. Please use ${project.version} instead.
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```

This is the reason why I've changed it. But can also revert it, if you believe this shouldn't be included!

Reply:
ok make sense , thanks for the change and the explanation